### PR TITLE
pageant for windows

### PR DESCRIPTION
**Pull request by abakum**

### CODE REVIEW

File: tssh/agent_windows.go [hunk 1]
```diff
@@ -29,11 +29,17 @@ import (
...
 
 	"github.com/Microsoft/go-winio"
+	"github.com/abakum/pageant"
 )
 
 const defaultAgentAddr = `\\.\pipe\openssh-ssh-agent`
 
 func dialAgent(addr string) (net.Conn, error) {
+
+	if rw, err := pageant.NewConn(); err == nil {
```
Comment (kke) [inline tssh/agent_windows.go:39]: Maybe something like:

```go
	if pageant.Available() {
		return pageant.New(), nil
	}
```

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,7 @@
 module github.com/trzsz/trzsz-ssh
 
-go 1.20
+go 1.21.4
```
Comment (lonnywong) [inline go.mod:3]: I use `go 1.20` for compatibility with Windows 7.

Comment (abakum) [reply go.mod:3]: > I use `go 1.20` for compatibility with Windows 7.
https://github.com/abakum/Win7

Comment (lonnywong) [reply go.mod:3]: https://github.com/golang/go/issues/57003

Comment (abakum) [reply go.mod:3]: "Αμφισβητήστε τα πάντα" \8^)

**Pull request closed without merging**